pie/storage/database.py

```diff
@@ -0,0 +1,70 @@
+from __future__ import annotations
+
+from typing import Dict, Optional, Union
+
+from sqlalchemy import BigInteger, Column, String
+
+from pie.database import database, session
+
+
+class Storage(database.base):
+    __tablename__ = "storage_data"
```

Comment:
We've recently started calling built-in tables `pie_*` (for tracking purposes), so this should be `pie_storage_data`.

Reply:
Gonna fix that.